checkstyle_test.xml

```diff
@@ -118,6 +118,13 @@
       <property name="message" value="Use the CassandraRelevantProperties or CassandraRelevantEnv instead." />
     </module>
 
+    <module name="RegexpSinglelineJava">
+      <property name="id" value="clearValueSystemPropertyUsage"/>
+      <property name="format" value="\.clearValue\("/>
+      <property name="ignoreComments" value="true"/>
+      <property name="message" value="Please use withProperties approach instead." />
```

Comment:
@bbotella please change the message to 

`Please use WithProperties in try-with-resources instead. See CASSANDRA-18453.`